src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
+const body = document.querySelector('body');
+const counter = document.createElement('div');
+body.append(counter);
 
-console.log('hello mate academy!');
+if (sessionStorage.getItem('counter') !== null) {
+  counter.textContent = sessionStorage.getItem('counter');
+}
+
+body.addEventListener('click', () => {
+  counter.textContent++;
```

Comment:
краще використай змінну для збільшення числа
```suggestion
  count++
  counter.textContent = count;
```

Reply:
А тут зовсім не знаю як(

Reply:
начебто done